server/.env

```diff
@@ -1,3 +1,4 @@
 TEAM_NAME="Jason Chui"
+TEAM_MEMBER="Philippe Charpentier"
```

Comment:
Sorry, this template didn't have great example: if you could add your name to the TEAM_NAME variable, just separate the names with commas